src/Tizen.NUI.Scene3D/src/public/Controls/Camera.cs

```diff
@@ -581,6 +662,38 @@ internal bool GetInvertYAxis()
             return ret;
         }
 
+        /// <summary>
```

Comment:
What is the use case of these method?
The input parameter verticalFov and horizontalFov are not the member variable..

Reply:
Thank you for remind me. It was some helper method to compute FoV.
It will be helpful if user want to use both Vertical FoV and Horizontal FoV.
Now I make it as public static method now.